db/structure.sql

```diff
@@ -28,15 +29,15 @@ CREATE TABLE `bmap_flowcell` (
   `recorded_at` datetime NOT NULL COMMENT 'Timestamp of warehouse update',
   `id_sample_tmp` int unsigned NOT NULL COMMENT 'Sample id, see "sample.id_sample_tmp"',
   `id_study_tmp` int unsigned NOT NULL COMMENT 'Study id, see "study.id_study_tmp"',
```

Comment:
Most of these changes are due to the frustratingly inconsistent ways in which different MySQL versions dump their table schemas.

Reply:
Ohh, should add, this won't affect the production warehouses at all. Just developer ones, and the CHARACTER SET option its dumped here matches what we have in production anyway.